src/RenderWebGL.js

```diff
@@ -58,6 +58,31 @@ const FENCE_WIDTH = 15;
 
 
 class RenderWebGL extends EventEmitter {
+    /**
+     * Check if this environment appears to support this renderer before attempting to create an instance.
+     * Catching an exception from the constructor is also a valid way to test for (lack of) support.
+     * @param {canvas} [optCanvas] - An optional canvas to use for the test. Otherwise a temporary canvas will be used.
+     * @returns {boolean} - True if this environment appears to support this renderer, false otherwise.
+     */
+    static isSupported (optCanvas) {
+        try {
+            // Create the context the same way that the constructor will: attributes may make the difference.
+            return !!RenderWebGL._getContext(optCanvas || document.createElement('canvas'));
+        } catch (e) {
+            return false;
+        }
+    }
+
+    /**
+     * Ask TWGL to create a rendering context with the attributes used by this renderer.
+     * @param {canvas} canvas - attach the context to this canvas.
+     * @returns {WebGLRenderingContext} - a TWGL rendering context (backed by either WebGL 1.0 or 2.0).
+     * @private
+     */
+    static _getContext (canvas) {
+        return twgl.getContext(canvas, {alpha: false, stencil: true});
```

Comment:
@cwillisf I notice that the twgl method here changed from `getWebGLContext` to `getContext`. Is there an important reason for that?

Reply:
Oops just looked at the docs and saw that this will create a webgl2 context if possible. I'm not sure we want to introduce that change here. If we want to support webgl 2, it might be a good idea to do that as a self contained unit so that we can see if there are any compatibility issues or things we can take advantage of.

Reply:
Hmm... that's fair. I'll change this back & file a lower-priority issue.